streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -3159,7 +3159,7 @@ private void addRecord(final MockConsumer<byte[], byte[]> mockConsumer,
     }
 
     StreamTask activeTask(final TaskManager taskManager, final TopicPartition partition) {
-        final Stream<Task> standbys = taskManager.allTasks().values().stream().filter(Task::isActive);
+        final Stream<Task> standbys = taskManager.allOwnedTasks().values().stream().filter(Task::isActive);
```

Comment:
For my understanding: why we need to change the callee here as well?

Reply:
Good point, we don't